pkg/ebpf/bpf_error.go

```diff
@@ -0,0 +1,223 @@
+package ebpf
+
+import (
+	"encoding/binary"
+	"fmt"
+	"math"
+	"strings"
+	"time"
+	"unsafe"
+
+	"github.com/aquasecurity/tracee/pkg/logger"
+)
+
+const BPF_MAX_ERR_FILE_LEN = 72
```

Comment:
We use CamleCase for consts in golang, even when they have matching value in the BPF code.
You should write it as CamleCase, and write comment that it matches the value in the BPF code.